low-level/files/etc/cron.hourly/sspl_logrotate

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
@saurabhkhanvilkar @cdeshmukh 
Avoid creating multiple custom files.
Use `https://github.com/Seagate/cortx-prvsnr/pull/622` to avoid any files under `/etc/cron.hourly` and just add files under `/etc/logrotate_hourly.d`